Header header
geometry_msgs/Vector3 vel
float32 yawRate